README.md

```diff
@@ -1280,6 +1279,7 @@ API | Description | Auth | HTTPS | CORS |
 API | Description | Auth | HTTPS | CORS |
 |---|---|---|---|---|
 | [apilayer screenshotlayer](https://screenshotlayer.com) | URL 2 Image | No | Yes | Unknown |
+| [APITemplate.io](https://apitemplate.io) | Dynamically generate images and PDFs from templates with a simple API  | `apiKey` | Yes | Yes |    
```

Comment:
```suggestion
| [APITemplate.io](https://apitemplate.io) | Dynamically generate images and PDFs from templates with a simple API | `apiKey` | Yes | Yes |    
```

Reply:
Each segment must start and end with exactly 1 space.

Reply:
Hi @pawelborkar Removed the extra white space
Thanks!